app/404.html

```diff
@@ -1,157 +1,113 @@
+    <link rel="icon" type="image/png" href="/images/favicons/android-chrome-192x192.png" sizes="192x192">
+    <link rel="icon" type="image/png" href="/images/favicons/favicon-96x96.png" sizes="96x96">
+    <link rel="icon" type="image/png" href="/images/favicons/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/images/favicons/manifest.json">
+    <meta name="msapplication-TileColor" content="#00aba9">
+    <meta name="msapplication-TileImage" content="/images/favicons/mstile-144x144.png">
+    <meta name="theme-color" content="#ffffff">
+    <!-- build:css(.tmp) styles/main.css -->
+    <link rel="stylesheet" href="styles/main.css">
+    <!-- endbuild -->
   </head>
-  <body>
-    <div class="container">
-      <h1>Not found <span>:(</span></h1>
-      <p>Sorry, but the page you were trying to view does not exist.</p>
-      <p>It looks like this was the result of either:</p>
-      <ul>
-        <li>a mistyped address</li>
-        <li>an out-of-date link</li>
-      </ul>
-      <script>
-        var GOOG_FIXURL_LANG = (navigator.language || '').slice(0,2),GOOG_FIXURL_SITE = location.host;
-      </script>
-      <script src="//linkhelp.clients.google.com/tbproxy/lh/wm/fixurl.js"></script>
+  <body ng-app="imls">
+    <!--[if lt IE 7]>
+      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+<div class="secondary-page">
+    <div class="navbar action-bar">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-xs-7">
+                    <div class="brand">
+                        <a class="logo" href="/">
+                            <img srcset="/images/2x/museum-stat-logo@2x.png 2x" src="/images/museum-stat-logo.png">
+                        </a>
+                    </div>
+                    <nav>
+                        <a href="/#/home">Home</a>
+                        <a href="/#/about">About</a>
+                        <a href="/#/help">Help</a>
+                        <a href="/#/contact">Contact</a>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <section class="main-content">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-xs-8 col-xs-push-2">
+                    <div class="content-title">
+                        <img class="content-title-icon" src="/images/museum-icon.svg">
+                        <h2>Page not found</h2>
+                        <hr>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse.</p>
```

Comment:
May want to get some actual 404 page text from @jbranigan or @designmatty before merging, so we don't push this and then forget about it.